app.js

```diff
@@ -1,8 +1,83 @@
 // import functions and grab DOM elements
+const buttonOne = document.getElementById('button-one');
+const buttonTwo = document.getElementById('button-two');
+const buttonThree = document.getElementById('button-three');
 
-// let state
+const cupOne = document.getElementById('cup-one-img');
+const cupTwo = document.getElementById('cup-two-img');
+const cupThree = document.getElementById('cup-three-img');
 
-// set event listeners 
-  // get user input
-  // use user input to update state 
-  // update DOM to reflect the new state
+const winsEl = document.getElementById('wins');
+const totalEl = document.getElementById('total');
+const lossesEl = document.getElementById('losses');
+
+// initial state, setting a let variable so it can be changed later on
+const randomCupNumber = [0, 1, 2];
+
```

Comment:
Glad this array worked out!  Sorry for all the confusion when we were working through how to approach this problem.